qemu/tests/cfg/virtio_chardev_trace.cfg

```diff
@@ -0,0 +1,11 @@
+- virtio_chardev_trace:
+    only Linux
+    type = virtio_chardev_trace
+    start_vm = no
+    smp = 4
```

Comment:
For the sake of insurance, it seems we need to check the CPU number of the host in case some machine has only 4 CPUs. And another way to resolve it is that we can set `smp=2`.